cron.yaml

```diff
@@ -0,0 +1,5 @@
+cron:
```

Comment:
agree with moving to an example file with comments. @transcedentalia mind doing the needful here?